contents/docs/integrate/webhooks/discord.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Discord
+sidebar: Docs
+showTitle: true
+---
+
+> For message formatting instructions, see [this dedicated page](/docs/integrate/webhooks/message-formatting).
+## 1. Create an incoming webhook in Discord 
+
+1. Navigate to the channel where you want to add the webhook and select 'Edit Channel' option from left navigation pane. 
+1. Select 'Integerations' option from the left navigation pane. 
+1. If creating a webhook for first time, click "Create Webhook" button. 
+1. If you have other webhooks, click "View Webhooks" and now click "New Webhook". 
+1. Give any name you prefer (say PostHog). 
+1. Your preferred channel would be appearing in 'Channel' option, If not select you preferred channel from drop-down menu.
+1. Click the "Copy Webhook URL" button to copy the webhook URL into the clipboard.  
+
+It would look something like this. 
```

Comment:
```suggestion
It would look something like this. 

For more information see the [Discord webhook docs](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks).
```